apps/minifront/src/state/swap/dutch-auction/index.ts

```diff
@@ -175,18 +179,19 @@ export const createDutchAuctionSlice = (): SliceCreator<DutchAuctionSlice> => (s
     }
   },
 
-  endAuction: async auctionId => {
-    const source = getAddressIndex.optional()(get().swap.assetIn);
-    const req = new TransactionPlannerRequest({ dutchAuctionEndActions: [{ auctionId }], source });
+  endAuction: async (auctionId, addressIndex) => {
+    const req = new TransactionPlannerRequest({
+      dutchAuctionEndActions: [{ auctionId }],
+      source: addressIndex,
```

Comment:
ack. passing in the `addressIndex` rather than fetching it from the swap information makes sense.